applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -53,6 +53,7 @@ static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_ISO_MAX_CHAN,
 static struct bt_audio_lc3_preset lc3_preset_nrf5340 = BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO;
 
 static atomic_t iso_tx_pool_alloc[CONFIG_BT_ISO_MAX_CHAN];
+static uint8_t boot_bond_num;
```

Comment:
Nitpick: probably we can remove prefix "boot" since it is not very related with boot up I think.